tfs/handler.py

```diff
@@ -187,19 +193,16 @@ def write_tfs(
         save_index: bool or string. Default: False
             If True, saves the index of the data_frame to a column identifiable by INDEX_ID.
             If string, it saves the index of the data_frame to a column named by string.
-        colwidth: Column width
+        colwidth: Column width, can not be smaller than MIN_COLUMN_WIDTH
+        headerswidth: Formats the header width for both, keys and values
     """
-    tfs_file_path = pathlib.Path(tfs_file_path) if isinstance(tfs_file_path, str) else tfs_file_path
+    tfs_file_path = pathlib.Path(tfs_file_path)
     _validate(data_frame, f"to be written in {tfs_file_path.absolute()}")
+    data_frame = data_frame.copy()  # as it might be changed
+    lalign_first = False
     if save_index:
-        if isinstance(save_index, str):  # save index into column by name given
-            idx_name = save_index
-        else:  # save index into column, which can be found by INDEX_ID
-            try:
-                idx_name = INDEX_ID + data_frame.index.name
-            except TypeError:
-                idx_name = INDEX_ID
-        data_frame.insert(0, idx_name, data_frame.index)
+        lalign_first = True
```

Comment:
As I understand it, this boolean is for left-aligning the first element or not?
If so, I would advise a slightly more verbose name to avoid digging in code to understand, `left_align_first_element` or so?

Reply:
done